us-02/us-02.py

```diff
@@ -38,6 +39,9 @@ def responseOption():
     #     print("Response Length = " + str(len(responseValue)))
     # else:
     #     print("Response Length = " + str(len(responseValue)))
+    
+    nltk_tokens = nltk.word_tokenize(responseValue)
```

Comment:
i think this will work for now since its a single reponse. however, i think later we should have the semantic analysis stuff in their own functions. that will  help make our code more modular.